Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py

```diff
@@ -51,8 +51,8 @@
 
 upgradeProperties[2017] = {
     '2017' : {
-        'Geom' : 'DB:Extended',
-        'GT' : 'auto:phase1_2017_realistic',
+        'Geom' : 'Extended2017dev',
+        'GT' : 'auto:phase1_2017_hcaldev',
```

Comment:
Yes, this sounds like it could cause great chaos if it gets into pre12. And for pre13/the release, I have the feeling that nothing called 'dev' should be in the then-production workflows.

It seems to me that `Extended2017dev` should be merged into the standard 2017 geometry for the release, and ideally after pre12 to make the tracker validation not more complicated than it is already. Does this dev geometry still need validation? It seems last significant changes were in May.